python/user.py

```diff
@@ -83,10 +83,14 @@ def load_user_expressions(path):
 
 The @qgsfunction decorator accepts the following arguments:
 
-:param args: Defines the number of arguments. With ``args='auto'`` the number
+:param args: Defines the number of arguments. With ``args='auto'`` the number of
              arguments will automatically be extracted from the signature.
+             With ``args=-1``, you can pass any number of arguments.
```

Comment:
I'm not sure about this. Better wording is welcome.

Reply:
* any number of arguements are accepted
* the number of arguments can be of variable length
?

Reply:
+1 to any number of arguements are accepted